src/App.js

```diff
@@ -513,7 +543,7 @@ function MapDiorama({ effects }) {
         const s = stones[i];
         tmpo.position.fromArray(s.position);
         tmpo.rotation.set(0, Math.random(), 0);
-        tmpo.scale.set(1, 0.1, 1);
+        tmpo.scale.set(1, 1, 1);
```

Comment:
Sorry I reverted this. I made them tall in the Y direction so that if they are placed on a slope they will still be in full contact with the ground instead of floating above. Though floating stones look cool too.

Reply:
I changed it to make the highlighting better. But I think we don't need that, I'll just color the stages that we've already won.